### PR TITLE
feat: create a launchpad homepage

### LINKED ISSUE #10: [Feature Request]: Create a launchpad home page
### Feature details

It will be great to create a launch pad home page and use that rather than having the default Next.js initial page.

### Suggested Solution

It can have the information that we have in the readme.. like:

- Purpose
- Tech involved
- How to get started.
- A footer talk about Open SOurce from CreoWis

It should be a very lightweight page without spending too much time on it.

### Affected areas

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by Rick799**
Created a launchpad homepage.

Fixes #10 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.

# Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

# Screenshots or example output

### CODE REVIEW

File: src/app/page.tsx [hunk 1]
```diff
@@ -1,113 +1,529 @@
...
-              src="/vercel.svg"
-              alt="Vercel Logo"
-              className="dark:invert"
-              width={100}
-              height={24}
-              priority
-            />
-          </a>
+    <>
+      <section className="w-full py-12  lg:py-14 xl:py-24 2xl:py-28">
```
Comment (joypoddar) [inline src/app/page.tsx:9]: There is a gap below the footer in the desktop view. `min-h-[94vh]` or `2xl:py-[7.4rem]` will solve the issue. Please check.

File: src/app/page.tsx [hunk 2]
```diff
@@ -1,113 +1,529 @@
...
+    name: 'TypeScript',
+    description:
+      "Enhance your development experience with TypeScript's static typing.",
+    link: 'https://www.typescriptlang.org/',
+    icon: ICONS.typescript,
+  },
+
+  {
+    id: 3,
+    name: 'TailWind CSS',
```
Comment (joypoddar) [inline src/app/page.tsx:448]: TailWind -> Tailwind

File: src/app/page.tsx [hunk 3]
```diff
@@ -1,113 +1,529 @@
...
+    description:
+      "Rapidly build custom designs with Tailwind CSS's utility-first approach.",
+    link: 'https://tailwindcss.com/',
+    icon: ICONS.tailwind,
+  },
+
+  {
+    id: 4,
+    name: 'ES Lint',
```
Comment (joypoddar) [inline src/app/page.tsx:457]: ES Lint -> ESLint

File: src/app/page.tsx [hunk 4]
```diff
@@ -1,113 +1,529 @@
...
-        </a>
-      </div>
-    </main>
+      </footer>
+    </>
   );
 }
+
+const ICONS = {
+  next_js: (
```
Comment (joypoddar) [inline src/app/page.tsx:58]: Remove unused icons

**Review by joypoddar — CHANGES_REQUESTED** [review]

**Comment by joypoddar** [comment]
@Rick799 Good work. Everything looks clean.

**Review by atapas — APPROVED** [review]

**Review by joypoddar — APPROVED** [review]

**Comment by atapas** [comment]
@all-contributors please add @Rick799  for Code. please add @joypoddar  for Code.

**Comment by atapas** [comment]
@all-contributors please add @joypoddar  for Code

**Comment by atapas** [comment]
@all-contributors please add @chhakuli123  for Code

**Comment by atapas** [comment]
@all-contributors please add @Jayakumar03 for Code

**Comment by atapas** [comment]
@all-contributors please add @atapas for Review

**Pull request merged**